docs/support-matrix.md

```diff
@@ -47,4 +48,5 @@ After you publish your plugin, open a PR that adds your plugin to the appropriat
 [9]: quickstart.md
 [10]: https://kubernetes.slack.com/messages/ark-dr
 [11]: https://github.com/heptio/ark/issues
-[12]: https://github.com/aws/aws-sdk-go/aws
\ No newline at end of file
+[12]: https://github.com/aws/aws-sdk-go/aws
+[13]: https://github.com/StackPointCloud/ark-plugin-digitalocean/blob/master/examples/10-ark-config.yaml#L15
```

Comment:
it seems brittle to me to link directly to source code here.  I think we should re-use the existing link (number 7)